src/components/Sidebar/SidebarNav.js

```diff
@@ -0,0 +1,46 @@
+import { Navbar, Nav } from 'react-bootstrap';
+import { useDispatch } from 'react-redux';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { toggleDeleteMode } from '../../redux/slices/carSlice';
+import carRental from './carRental.png';
+import 'boxicons';
+import './sidebar.css';
+
+function Sidebar() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Sidebar= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪